### ISSUE TITLE
Examples not rendering in Altair's example gallery

### ISSUE THREAD

**Issue by louis-vines** [altair-viz/altair#986]
Hi, having recently switched to using Python for work and coming from an R background I've been really keen to start using Altair as it seems to be the closest python library to ggplot.

However quite a few of the examples aren't rendering in the examples gallery in my browser. For example, the simple highlighted bar chart:

https://altair-viz.github.io/gallery/simple_highlighted_bar_chart.html

here is screen grab of the page:

![image](https://user-images.githubusercontent.com/10738810/42117427-44e94366-7bf4-11e8-801b-0dd86ff7ebf2.png)

There are plenty of other examples that are exhibiting the same behaviour.

Having read a bit of the docs, this behaviour seems to be consistent with a chart that's incorrectly specified which would imply that the examples are out of date with the current version of altair/vega-lite. Or alternatively something is wrong with my setup, since these pages are already rendered this implies it must be a compatability issue with my browser - I'm running an up to date version of Google Chrome (Version 67.0.3396.87).

Any insight into this would be much appreciated as I'm quite keen to start adopting Altair into my workflows!

**Comment by louis-vines** [comment]
Actually having looked into this a bit, I've found the problem - the current version of Altair doesn't seem to be able to accept URLs pointing to json. All the examples in the example gallery that use the `data.<dataset>.url` approach to pointing to the data seem to be failing.

If I change line 4 in the example above to:

```
population = data.population()
```
it works totally fine.

**Comment by jakevdp** [comment]
All the examples work fine for me, and I'm also on Chrome 67.0.3396.

Do you have adblock turned on? Sometimes that prevents javascript from accessing external URLs, which is important in the case of Altair.

**Comment by lincolnfrias** [comment]
I had a similar problem. The solution was to update the vega_datasets package.

**Comment by jakevdp** [comment]
> I had a similar problem. The solution was to update the vega_datasets package.

I don't think that would have any effect on the rendering of charts on the website.

**Comment by jakevdp** [comment]
@louis-vines were you able to check whether adblock affects this, and whether there were helpful errors in the javascript console? Thanks

**Comment by gwdekker** [comment]
Confirming @jakevdp hypothesis for my case: I had the same problem, which got fixed by disabling my adblocker. I propose to add this to the FAQ section of the documentation.

**Comment by FlorianWetschoreck** [comment]
The issue was fixed in the course of #1199 and can be closed

**Comment by louis-vines** [comment]
Really sorry for some reason I wasn't getting pinged for this issue. You are correct I did have an adblocker on. It seems to work now with the adblocker on I guess you guys resolved this with #1199 
.

Thanks!
